pkg/flagenv/flagenv.go

```diff
@@ -8,16 +8,16 @@
 // This example will make it possible to set the default of --my_flag also via the MY_PROG_MY_FLAG
 // env var:
 //
-//    var myflag = flag.String("my_flag", "", "some flag")
+//	var myflag = flag.String("my_flag", "", "some flag")
```

Comment:
Is this a tabs vs spaces thing?
why dd this change?

Reply:
This was automatic formatting by `goimports`, which is one of the linting tools we use.

Reply:
ok